CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+ - #3504, Eliminate `format` parameter on `rowFilter` parameters in OpenAPI - @dantheman2865
```

Comment:
```suggestion
 - #3504, Remove `format` from `rowFilter` parameters in OpenAPI - @dantheman2865
```

This was OK under "Fixed".

Reply:
I pushed a incorporating this feedback and moving the message back to "Fixed". Thanks!